packages/backend-plugin-manager/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@backstage/backend-plugin-manager",
```

Comment:
TBF I was expecting it would be published, so that we can easily use it in some external backstage application repository, without the need to be in the main `backstage/backstage` repository.
Isn't the fact that there's an clear experimental warning in the `Readme`  sufficient, as it was for the [`backend-plugin-api`](https://www.npmjs.com/package/@backstage/backend-plugin-api?activeTab=readme) ?

Reply:
Also in the Experimental `backend-app-api`, there were some breaking changes on an already published `public` API defined as Experimental in the package Readme:  https://github.com/backstage/backstage/blob/master/packages/backend-app-api/CHANGELOG.md#minor-changes

Reply:
It's a way to get it merged without more thorough review and considerations for API design etc. In the case of the backend system we had already been iterating on it for quite a while in a common branch so we went with the strategy of shipping the packages straight away. For the frontend system we've merged the initial work much earlier, so those packages are currently [marked as private](https://github.com/backstage/backstage/blob/535f5fd6ce466baf29f2c3a3c2d06b9c9c80dca3/packages/frontend-app-api/package.json#L7). By marking them as private we can make a lot more smaller iterations and defer the final review until we want to publish the packages. While we ofc make followup breaking changes we don't want to publish things that we know will break in the future if it is easy to avoid.

A fairly straightforward way to try out the package in a different project is to `yarn build && yarn pack` the package and then install the generated tarball as a dependency in the other project using a `file:` dependency.

If you want to get it published straight away we can instead continue with that in mind, let me know. I wanted to suggest this path as a way of simplifying and speeding up iteration, but it's not required.

Reply:
> It's a way to get it merged without more thorough review and considerations for API design etc. In the case of the backend system we had already been iterating on it for quite a while in a common branch so we went with the strategy of shipping the packages straight away. For the frontend system we've merged the initial work much earlier, so those packages are currently [marked as private](https://github.com/backstage/backstage/blob/535f5fd6ce466baf29f2c3a3c2d06b9c9c80dca3/packages/frontend-app-api/package.json#L7). By marking them as private we can make a lot more smaller iterations and defer the final review until we want to publish the packages. While we ofc make followup breaking changes we don't want to publish things that we know will break in the future if it is easy to avoid.
> 

Thanks for the context. I understand the difference.

> A fairly straightforward way to try out the package in a different project is to `yarn build && yarn pack` the package and then install the generated tarball as a dependency in the other project using a `file:` dependency.
> 

Sure.

> If you want to get it published straight away we can instead continue with that in mind, let me know. I wanted to suggest this path as a way of simplifying and speeding up iteration, but it's not required, sorry if it came across that way.

Let's make it private then, and iterate.